sources/web/datalab/polymer/components/datalab-settings/datalab-settings.html

```diff
@@ -14,21 +14,55 @@
 
 <link rel="import" href="../../components/shared-styles/shared-styles.html">
 
-<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/neon-animation/web-animations.html">
 <link rel="import" href="../../bower_components/paper-input/paper-input.html">
+<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/paper-tooltip/paper-tooltip.html">
+<link rel="import" href="../../bower_components/paper-progress/paper-progress.html">
 
 <dom-module id="datalab-settings">
   <template>
     <style include="datalab-shared-styles">
+      paper-progress {
+        width: 100%;
+        height: 1px;
+        --paper-progress-active-color: var(--selection-fg-color);
+        --paper-progress-container-color: var(--border-color);
+      }
       paper-radio-button {
         --paper-radio-button-checked-color: var(--selection-fg-color);
       }
       .setting {
         padding: 15px;
         border-bottom: 1px solid var(--border-color);
       }
+      .input-container {
+        display: flex;
+        height: 35px;
+      }
+      .input-container > * {
+        margin-right: 10px;
+      }
+      .input-box {
+        width: 150px;
+        --primary-text-color: var(--primary-fg-color);
+      }
+      .super-info {
+        width: 18px;
+        height: 18px;
+        color: #888;
+        padding: 0px;
+      }
+      #idleTimeoutTooltip {
+        --paper-tooltip: {
+          font-size: 13px;
+        }
+      }
     </style>
 
+    <!--Thin progress bar that shows until files are loaded-->
+    <paper-progress id="progressBar" indeterminate disabled={{!_busy}}></paper-progress>
```

Comment:
It's only a busy indicator, but I'm using this element to match the style used in other elements. I think it looks better than a spinning circle, which is the other popular element.